lib/rack/auth/digest/params.rb

```diff
@@ -17,7 +17,7 @@ def self.dequote(str) # From WEBrick::HTTPUtils
         end
 
         def self.split_header_value(str)
-          str.scan( /(\w+\=(?:"[^\"]+"|[^,]+))/n ).collect{ |v| v[0] }
+          str.scan(/\w+\=(?:"[^\"]+"|[^,]+)/)
```

Comment:
Sorry, I have meas-understood this, I thought it means different - http://www.regular-expressions.info/modifiers.html.
Will return it back.
Thanks